helm-chart/renku/values.yaml

```diff
@@ -171,21 +171,34 @@ ingress:
   #      - example.local
 
 ## Keycloak configuration
-keycloak:
+keycloakx:
```

Comment:
There are already a few checks in `_helpers.tpl` that fail on specific conditions with an explanation why. We should add a similar statement that will fail the helm upgrade/install if we detect that the values file does not have a `keycloakx` field but has a `keycloak` field. I feel like people may not read the release notes and not upgrade the values file but simply try to install the helm chart and then wonder why everything is broken. What do you think?